tools/mo/openvino/tools/mo/back/offline_transformations.py

```diff
@@ -104,7 +59,28 @@ def read_model(path_to_xml):
 
     func = read_model(input_model + "_tmp.xml")
 
-    add_layouts(func, argv)  # TODO: replace with preprocessing
+    reverse_input_channels = False
+    if 'reverse_input_channels' in argv:
```

Comment:
please add TODO that we reset this options temporarily. Now we still use legacy preprocessing